codebuild/release/release-prod.yml

```diff
@@ -0,0 +1,86 @@
+version: 0.2
+# 'release-prod' is an AWS::CodeBuild::BuildSpec that Releases to Sonatype and then validates the release with 'validate-prod'
+# Command to trigger this codebuild from an authorized command line.
+#  aws codebuild start-build-batch \
+#  --region us-west-2 \
+#  --project-name java-ddb-ec-release \
+#  --environment-variables-override name=VERSION,value=<INSERT_VERSION>,type=PLAINTEXT name=COMMIT_ID,value=<INSERT_COMMIT_ID>,type=PLAINTEXT"
+
+env:
+  secrets-manager:
+    GPG_KEY: Maven-GPG-Keys-Credentials:Keyname
+    GPG_PASS: Maven-GPG-Keys-Credentials:Passphrase
+    SONA_USERNAME: Sonatype-Team-Account:Username 
+    SONA_PASSWORD: Sonatype-Team-Account:Password
+
+phases:
+  install:
+    runtime-versions:
+      java: openjdk8
+  pre_build:
+    commands:
+      - git checkout $COMMIT_ID
+      - FOUND_VERSION=$(grep version sdk1/pom.xml | head -n 2 | sed -n 's/[ \t]*<version>\(.*\)<\/version>/\1/p')
+      - |
+        if expr ${FOUND_VERSION} != ${VERSION}; then
+          echo "pom.xml version (${FOUND_VERSION}) does not match expected version (${VERSION}), stopping"
+          exit 1;
+        fi
+      - export SETTINGS_FILE=$(pwd)/codebuild/release/settings.xml
+      - aws secretsmanager get-secret-value --region us-west-2 --secret-id Maven-GPG-Keys --query SecretBinary --output text | base64 -d > ~/mvn_gpg.tgz
+      - tar -xvf ~/mvn_gpg.tgz -C ~
+  build:
+    commands:
+      - |
+        mvn deploy \
+          -Ppublishing \
+          -DperformRelease \
+          -Dgpg.homedir="$HOME/mvn_gpg" \
+          -DautoReleaseAfterClose=true \
+          -Dgpg.keyname="$GPG_KEY" \
+          -Dgpg.passphrase="$GPG_PASS" \
+          -Dsonatype.username="$SONA_USERNAME" \
+          -Dsonatype.password="$SONA_PASSWORD" \
+          -s $SETTINGS_FILE
+
+
+batch:
+  fast-fail: false
+  build-graph:
+    - identifier: release_to_prod
```

Comment:
this step should have the buildspec file path, no?

Reply:
I do not think so. This is referring to itself as the first step in the build batch

Reply:
gotcha!

Reply:
I generated this PR by verbatim copying the `codebuild` directory from 2.x into 1.x. Since we have successfully deployed 2.x with these buildspecs, I am inclined to trust that they are correct. I will double check with the other repos.

Reply:
The Python ESDK does the same thing: https://github.com/aws/aws-encryption-sdk-python/blob/0922a76eb4536cbc2246e723f97496e068204d78/codebuild/release/prod-release.yml#L34